index.html

```diff
@@ -0,0 +1,314 @@
+                        <div class="logo_title">
+                            <div class="logo_title1">CM</div>
+                            <div class="logo_title2">Summit 2015</div>
+                        </div>
+                    </div>
+                </a>
+                <div class="menu">
+                    <!-- <span clas="close">&times;</span> -->
+                    <ul class="navlist">
+                        <li> <a href="about.html" class="nav_link">About</a></li>
+                        <li> <a href="" class="nav_link">Program</a></li>
+                        <li> <a href="" class="nav_link">Join</a></li>
+                        <li> <a href="" class="nav_link">Sponsor</a></li>
+                        <li> <a href="" class="nav_link">News</a></li>
+                        <li> <a href="" class="nav_link"><button class="home_button">CM Campain</button></a></li>
+                    </ul>
+                </div>  
+            </nav>
+
+            <div class="menu-hide" id="get_menu">
+                <button id="close" clas="close">&times;</button> 
+                <ul class="menulist">
+                    <li> <a href="index.html" class="menu_link">Home</a></li>
+                    <li> <a href="about.html" class="menu_link">About</a></li>
+                    <li> <a href="" class="menu_link">Program</a></li>
+                    <li> <a href="" class="menu_link">Join</a></li>
+                    <li> <a href="" class="menu_link">Sponsor</a></li>
+                    <li> <a href="" class="menu_link">News</a></li>
+                    <li> <a href="" class="menu_link"><button class="home_button">CM Campain</button></a></li>
+                </ul>
+            </div>  
+
+        </header>
+       
+        <section class="title-section">
+           
+                <div class="hamb">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            
+            <h2 class="first_head">
+                <q>Hello! Sharing world</q>
+            </h2>
+            
+            <!--<h1 class="second_head">KAMPALA PROGRAMMERS FORUM 2023</h1>-->
+            <h1 class="second_head">CODING MINDS <br>INTERNATIONAL SUMMIT <br>2015</h1>
+            <br>
+            <article class="info">
+                A joyful celebration believing in the value of openess and sharing, creating a positive change with
+                people from over 80 countries is taking place in October, in Kampala
+            </article>
+            <br>
+            <time class="date">
+                2015.10.15(THU) ~ 16(FRI)
+            </time>
+
+            <P class="venue">@National Museum of Uganda, Art Center Nabi and more</P>
+        </section>
+
+
+        <section class="program">
+            <h2 class="prog_heading">
+                Main Program
+            </h2>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add `orange line` under to `heading` like a [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/219430301-4c8fc2ad-867a-4524-87d3-0a1d4c0bcf56.png)